src/app/issues/page.tsx

```diff
@@ -31,16 +31,15 @@ interface IIssueData {
 }
 
 export default function IssuesPage({}) {
-  const [issueType, setIssueType] = useState<
-    "created" | "assigned" | "mentioned"
-  >("created");
-
-  const [search, setSearch] = useState<string>(
-    `is:open is:issue author:xxxxx archived:false`
-  );
   const [issues, setIssues] = useState<IIssueData[]>(openData);
   const [issueStatus, setIssueStatus] = useState<"open" | "closed">("open");

```

Comment:
The name setIssueStatus is confusing to me, because it could easily be interpreted as "as a maintainer, set the status of a particular issue to open or closed". But in reality it determines the tab which is currently shown (i.e. filter issues based on the issue state). I suggest the names be changed to [issuesFilter, setIssuesFilter].

Reply:
Or maybe [issuesTab, setIssuesTab] because we have a separate filter input field which should not be confused with this.

Reply:
Or am I totally confused about the issue? :smile: @pierresisson does this make sense to you?